server/routes.go

```diff
@@ -265,6 +270,10 @@ func (s *RESTServer) addRoutes() http.Handler {
 		{"GET", "/upload/:fileid/metadata", RoleMDOnly, s.GetFileInfoHandler},
 		{"PUT", "/upload/:fileid/metadata", RoleWrite, s.SetFileInfoHandler},
 
+		// /admin/tape_use (enable, disable, get status)
+		{"GET", "/admin/use_tape", RoleWrite, s.GetTapeUseHandler},
+		{"PUT", "/admin/use_tape/:status", RoleWrite, s.SetTapeUseHandler},
```

Comment:
well, decide which you prefer

On Thu, Aug 25, 2016 at 12:36 PM, Don Brower <notifications@github.com>
wrote:

> In server/routes.go
> <https://github.com/ndlib/bendo/pull/136#discussion_r76278908>:
>
> > @@ -265,6 +270,10 @@ func (s *RESTServer) addRoutes() http.Handler {
> >  		{"GET", "/upload/:fileid/metadata", RoleMDOnly, s.GetFileInfoHandler},
> >  		{"PUT", "/upload/:fileid/metadata", RoleWrite, s.SetFileInfoHandler},
> >
> > +		// /admin/tape_use (enable, disable, get status)
> > +		{"GET", "/admin/use_tape", RoleWrite, s.GetTapeUseHandler},
> > +		{"PUT", "/admin/use_tape/:status", RoleWrite, s.SetTapeUseHandler},
>
> Make the route /admin/use_tape and then the handler will check for a
> status parameter.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ndlib/bendo/pull/136/files/116b45889c785d5598950eba80d6730d33f98945#r76278908>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADNGTH1ujr8JDU8xvHhLUxOQlETxlcSVks5qjcSdgaJpZM4JsckT>
> .
>